CHANGE_LOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Added WebChat banner on chat disconnect depending on visibility events
+- Added `sessionStorage` support to chat widget, exposing a broadcast event `RemoveWidgetDataFromCache` to help removing cache data
```

Comment:
Add a "Fixed" section and note down all the issues fixed by this PR. There should be multiple